src/main/java/frc/team2412/robot/commands/index/IndexSpitCommand.java

```diff
@@ -0,0 +1,27 @@
+package frc.team2412.robot.commands.index;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.team2412.robot.subsystem.IndexSubsystem;
+
+public class IndexSpitCommand extends CommandBase {
+    private final IndexSubsystem subsystem;
+
+    public IndexSpitCommand(IndexSubsystem subsystem) {
+        this.subsystem = subsystem;
+        addRequirements(subsystem);
+    }
+
+    @Override
+    public void initialize() {
+        // turn on both motors
+        subsystem.ingestMotorOut();
+        subsystem.feederMotorOut();
+    }
+
+    @Override
+    public boolean isFinished() {
+        subsystem.ingestMotorStop();
+        subsystem.feederMotorStop();
```

Comment:
Delete - this is just checking if the command is done and shouldn't be changing the motors

Reply:
If you want to stop the motors when the command ends, add that in a different method:
```
    @Override
    public void end(boolean interrupted) {
        ...
    }
````